### PR TITLE
feature/array_implementation change eslint errors in index.js after a…

### PR DESCRIPTION
**Pull request by ZelikSV**
add MyArray to index.js and fixed errors

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Nice job! Keep it up!

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,118 @@
...
+    delete this[this.length - 1];
+
+    this.length -= this.length;
+  }
+
+  forEach(callback) {
+    const mas = this;
+
+    for (let i = 0; i < mas.length; i++) {
+      callback(null, mas[i], i, mas);
```
Comment (Vladyslav-Shvorobuk) [inline src/index.js:28]: null not needed

File: src/tests/constructor.test.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ describe('tests for constructor', () => {
   test('should have own property length', () => {
     const arr = new MyArray();
 
-    expect(arr.Object.prototype.hasOwnProperty.call('length')).toBeTruthy();
+    expect(Object.prototype(arr, hasOwnProperty.call('length'))).toBeTruthy();
```
Comment (Vladyslav-Shvorobuk) [inline src/tests/constructor.test.js]: not right: Object.prototype(arr, hasOwnProperty.call('length')
must be Object.prototype.hasOwnProperty.call(arr, 'length')

**Review by Vladyslav-Shvorobuk — APPROVED** [review]
well done.

**Review by Arkant — APPROVED** [review]
It's fine

**Pull request merged**